packages/react-tinacms-editor/README.md

```diff
@@ -135,7 +136,27 @@ export function Page(props) {
 }
 ```
 
-### Example: Dynamic Imports
+### Sticky Menu
+
+When editing long content it is likely that the editor will scroll past the wysiwyg menu.
+
+By adding the `sticky` property the menu will follow the user as they scroll through the page.
```

Comment:
According to the docs sticky is the default, right?

Reply:
Nope, default is non-sticky